.github/workflows/build_and_test.yml

```diff
@@ -10291,444 +10291,937 @@ jobs:
-#        <command name="print_make_version"/>
-#        <command name="check_compiler" options="gcc"/>
-#        <command name="print_perl_version"/>
-#        <command name="print_autobuild_config"/>
-#        <command name="auto_run_tests" options="script_path=tests dir=$GITHUB_WORKSPACE/OpenDDS -Config XERCES3 -Config OPENDDS_SECURITY -Config CXX11 -Config RAPIDJSON -Config NO_SHMEM -Config DDS_NO_ORBSVCS --security"/>
-#        <command name="log" options="off"/>
-#        <command name="process_logs" options="prettify"/>
-#        </autobuild>
-#        EOF
-#        $GITHUB_WORKSPACE/autobuild/autobuild.pl "$GITHUB_WORKSPACE/OpenDDS/${{ github.job }}_autobuild_workspace/config.xml"
-#    - name: upload autobuild output
-#      uses: actions/upload-artifact@v3
-#      with:
-#        name: ${{ github.job }}_autobuild_output
-#        path: OpenDDS/${{ github.job }}_autobuild_workspace
-#    - name: check results
-#      shell: bash
-#      run: |
-#        $GITHUB_WORKSPACE/OpenDDS/tools/scripts/autobuild_brief_html_to_text.pl "$GITHUB_WORKSPACE/OpenDDS/${{ github.job }}_autobuild_workspace/output.log_Brief.html"
-#        cat "$GITHUB_WORKSPACE/OpenDDS/${{ github.job }}_autobuild_workspace/latest.txt"
-#        grep -q 'Failures: 0' "$GITHUB_WORKSPACE/OpenDDS/${{ github.job }}_autobuild_workspace/latest.txt"
+  #   steps:
+  #   - name: install xerces-c
+  #     run: |
+  #       brew install xerces-c
+  #   - name: checkout MPC
+  #     uses: actions/checkout@v3
+  #     with:
+  #       repository: DOCGroup/MPC
+  #       path: MPC
+  #   - name: checkout autobuild
+  #     uses: actions/checkout@v3
+  #     with:
+  #       repository: DOCGroup/autobuild
+  #       path: autobuild
+  #   - name: checkout ACE_TAO
+  #     uses: actions/checkout@v3
+  #     with:
+  #       repository: DOCGroup/ACE_TAO
+  #       ref: ace6tao2
+  #       path: ACE_TAO
+  #   - name: download ACE_TAO artifact
+  #     uses: actions/download-artifact@v3
+  #     with:
+  #       name: ACE_TAO_m11_sec_artifact
+  #       path: ACE_TAO
+  #   - name: extract ACE_TAO artifact
+  #     shell: bash
+  #     run: |
+  #       cd ACE_TAO
+  #       tar xvfJ ACE_TAO_m11_sec.tar.xz
+  #   - name: checkout OpenDDS
+  #     uses: actions/checkout@v3
+  #     with:
+  #       path: OpenDDS
+  #       submodules: true
+  #   - name: download OpenDDS artifact
+  #     uses: actions/download-artifact@v3
+  #     with:
+  #       name: build_m11_sec_artifact
+  #       path: OpenDDS
+  #   - name: extract OpenDDS artifact
+  #     shell: bash
+  #     run: |
+  #       cd OpenDDS
+  #       tar xvfJ build_m11_sec.tar.xz
+  #   - name: check build configuration
+  #     shell: bash
+  #     run: |
+  #       cd OpenDDS
+  #       . setenv.sh
+  #       tools/scripts/show_build_config.pl
+  #   - name: disable ntp
+  #     shell: bash
+  #     run: |
+  #       sudo systemsetup -setusingnetworktime off
+  #       sudo rm -rf /etc/ntp.conf
```

Comment:
What happens if NTP is enabled?

Reply:
I'm not sure. I brought that change over from m10 builds because I thought it might also be a problem on m11.
https://github.com/objectcomputing/OpenDDS/pull/2904 is the PR in which it was added to m10.

Reply:
Some of the unit tests were failing on macos builds due to ACE timers not firing at the right times. There had been a number of GitHub issues surrounding time & timers in the the macos GitHub virtual environments, and this was one suggested change. It didn't wind up having much impact on our tests, so we can probably remove it now.

Reply:
Ok, will do.